src/makeCalculator.js

```diff
@@ -3,8 +3,49 @@
 /**
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculatorObj = {
+    result: 0,
+
+    add(a) {
+      this.result += a;
+
+      return this;
+    },
+
+    subtract(a) {
+      this.result -= a;
+
+      return this;
+    },
+
+    multiply(a) {
+      this.result *= a;
+
+      return this;
+    },
+
+    divide(a) {
+      this.result /= a;
```

Comment:
It is necessary to add a check for 0, since it is not possible to divide by 0 according to the rules of mathematics